docs/release-notes/whats-new-3.rst

```diff
@@ -21,8 +21,45 @@ Imports
 + Put your shit here from v2                         | Put your shit here from v3                                             |
 +----------------------------------------------------+------------------------------------------------------------------------+
 
+OpenAPI Controller Replaced by Plugins
+--------------------------------------
```

Comment:
```suggestion
--------------------------------------

:doc:`OpenAPI v2 Reference Docs <litestar-v2:reference/openapi/openapi>`
```

It would be nice to be able to link to these. We could add an `intersphinx` entry for `litestar-v2`, but any of my attempts here have failed

<details>
<summary>Details</summary>

```
:param:`litestar-v2:litestar.openapi.config.OpenAPIConfig.enabled_endpoints`
:paramref:`litestar-v2:litestar.openapi.config.OpenAPIConfig.enabled_endpoints`
:paramref:`litestar-v2:~litestar.openapi.config.OpenAPIConfig.enabled_endpoints`
:paramref:`litestar-v2:openapi.config.OpenAPIConfig.enabled_endpoints`
:paramref:`litestar-v2:~.openapi.config.OpenAPIConfig.enabled_endpoints`
:class:`litestar-v2:~.openapi.config.OpenAPIConfig`
:class:`litestar-v2:~.openapi.config.OpenAPIConfig.enabled_endpoints`
:class:`litestar-v2:OpenAPIConfig`
:ref:`litestar-v2:OpenAPIConfig`
:doc:`litestar-v2:OpenAPIConfig`
```

</details>

Maybe we could just link into the API ref docs in general since maybe .inv files dont have what i think they should? I feel like it would be helpful at least in jumping to see what was in v2, etc. Happy to not do this, too.

```
:doc:`OpenAPI v2 Reference Docs <litestar-v2:reference/openapi/index>`
```
```
intersphinx_mapping = {
    ...
    "litestar-v2": ("https://docs.litestar.dev/2/", None),
}
```
would be needed if we did that

Reply:
Would you mind if we track this as a separate issue?

Reply:
fine with me for sure